### PR TITLE
Update transaction accept preview heading style, add line above footer

### PR DESCRIPTION
**Pull request by daniellemaxwell**
### Scope of changes

This updates the transaction accept preview page's heading to add styling to make it consistent with the transaction detail page. A horizontal line is also added to separate the body and footer.

### Type of change

- [ ] bug fix
- [ ] new feature
- [ ] documentation
- [x] other (describe)

### Acceptance criteria

https://www.awesomescreenshot.com/image/48286365?key=080b60fc0395ca5842e9d8f028681e89

### Author checklist

- [x] I have manually tested the change and/or added automation in the form of unit tests or integration tests
- [ ] I have updated the dependencies list
- [ ] I have added new test fixtures as needed to support added tests
- [ ] I have added or updated the documentation
- [x] Check this box if a reviewer can merge this pull request after approval (leave it unchecked if you want to do it yourself)

### Reviewer(s) checklist

- [ ] Any new user-facing content that has been added for this PR has been QA'ed to ensure correct grammar, spelling, and understandability.
- [ ] To the best of my ability, I believe that this PR represents a good solution to the specified problem and that it should be merged into the main code base.

### CODE REVIEW

File: pkg/web/templates/partials/transaction/transaction_accept.html [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <section>
-  <section class="lg:flex lg:justify-between lg:items-center">
-    <div class="pb-4 flex items-center gap-x-1 lg:pb-0">
-      <dt class="font-bold text-lg">Transaction ID:</dt>
-      <dd>{{ .ID }}</dd>
+  <section class="p-4 border border-black bg-[#D9D9D9] lg:flex lg:justify-between lg:items-center">
```
Comment (bbengfort) [inline pkg/web/templates/partials/transaction/transaction_accept.html:2]: Should this section also have rounded borders like the section below?

File: pkg/web/templates/partials/transaction/transaction_accept.html [hunk 2]
```diff
@@ -12,8 +12,8 @@
       <a href="/transactions" class="p-1 rounded w-32 bg-black font-semibold text-white text-center md:p-2 hover:bg-black/80">Back</a>
     </div>
   </section>
-  <section class="my-4">
-    <div class="mb-2 bg-[#E7EAEC] p-2 rounded">
+  <section class="my-8">
+    <div class="mb-2 bg-[#D9D9D9] p-2 rounded">
```
Comment (bbengfort) [inline pkg/web/templates/partials/transaction/transaction_accept.html:16]: Out of curiosity where did the #D9D9D9 gray color come from? Figma? It's feeling a little too dark for me, can I recommend #E3F5FB (a very light white blue) for a simpler contrast?

**Review by bbengfort — APPROVED** [review]
Thanks for making these style changes!

File: pkg/web/templates/partials/transaction/transaction_accept.html (hunk 1 above)
Comment (daniellemaxwell) [reply pkg/web/templates/partials/transaction/transaction_accept.html:2]: This style originally came from the preview modal and the border wasn't rounded there. I can update this to match the sections below.

File: pkg/web/templates/partials/transaction/transaction_accept.html (hunk 2 above)
Comment (daniellemaxwell) [reply pkg/web/templates/partials/transaction/transaction_accept.html:16]: Yes, #D9D9D9 came from Figma. How about #DCF0F9? It's still a light blue, but is a little darker against the white background. Here's a link to how it looks: https://tinyurl.com/225yastd

Comment (bbengfort) [reply pkg/web/templates/partials/transaction/transaction_accept.html:16]: Hmm, a little too icy for me. Let's stick with the gray and think about colors later for the MVP?

**Pull request merged**